messaging/src/test/java/org/axonframework/messaging/MessageStreamTest.java

```diff
@@ -49,7 +49,22 @@ public abstract class MessageStreamTest<M extends Message<?>> {
      *                 {@link MessageStream stream} under construction.
      * @return A {@link MessageStream stream} to use for testing.
      */
-    abstract MessageStream<M> testSubject(List<M> messages);
+    abstract MessageStream<M> completedTestSubject(List<M> messages);
+
+    /**
+     * Construct a test subject using the given {@code messages} as the source. The resulting stream should not report
+     * as completed if given messages are consumed.
+     * <p>
+     * It is the task of the implementer of this method to map the {@code messages} to {@link Entry entires} for the
```

Comment:
```suggestion
     * It is the task of the implementer of this method to map the {@code messages} to {@link Entry entries} for the
```

Reply:
I see this is a typo that exists in the method above, too. Likely my bad :sweat_smile: